src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,69 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturerAudi = new Manufacturer("Audi", "Germany");
+        Manufacturer manufacturerBmw = new Manufacturer("BMW", "Germany");
+        manufacturerService.create(manufacturerAudi);
+        manufacturerService.create(manufacturerBmw);
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
```

Comment:
services should be a class level variables

Reply:
not fixed